web/ui/module/lezer-promql/README.md

```diff
@@ -0,0 +1,44 @@
+# lezer-promql
+
+[![CircleCI](https://circleci.com/gh/promlabs/lezer-promql.svg?style=shield)](https://circleci.com/gh/promlabs/lezer-promql) [![NPM version](https://img.shields.io/npm/v/lezer-promql.svg)](https://www.npmjs.org/package/lezer-promql)
```

Comment:
These will have to be adapted to new CI locations. Or are you planning on doing that later?

Reply:
Oh indeed I totally forgot about it. For the moment I have no idea about this part. I'm even wondering how it will be integrated to codemirror-promql.
Since codemirror-promql is a published package it cannot depend on a local package. So I guess lezer-promql will be directly integrated in codemirror-promql.
@juliusv Does it sound good for you ?

Reply:
Yeah, that sounds good to me for now. If people want to consume `lezer-promql` separately (unlikely), they can still do it via the `codemirror-promql` package if we export it properly.